eslint.config.mjs

```diff
@@ -53,7 +53,8 @@ export const tsConfig = typescriptEslint.config({
     '@angular-eslint/no-output-native': ['off'],
     '@angular-eslint/directive-class-suffix': ['off'],
     '@angular-eslint/no-conflicting-lifecycle': ['off'],
-    '@angular-eslint/prefer-output-emitter-ref': ['off']
+    '@angular-eslint/prefer-output-emitter-ref': ['off'],
+    '@angular-eslint/no-developer-preview': ['off']
```

Comment:
I was also thinking of simply ignoring the one place where we use `effect`, but considering that we migrate everything to signals and might not want to upgrade to Angular 20 too soon, let's just globally disable the rule for now.